src/ui/easterEggs.js

```diff
@@ -47,9 +48,13 @@ export default {
         }
 
         goosemodScope.showToast(e.message);
+        
+        if (e.audio) {
+          new Audio(e.audio).play();
```

Comment:
Just leaving this as `new Audio(...).play()` results in a memory leak.

Reply:
Best to store the `new Audio(...)` play it, then after it's played remove the element.

Reply:
Resolved by latest PR commit.